nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/server/CommonClassLoaderServiceImpl.java

```diff
@@ -168,7 +169,7 @@ private void createCommonClassLoader() {
                     urls.toArray(new URL[urls.size()]), APIClassLoader);
             commonClassLoader.enableCurrentBeforeParent();
         } else {
-            logger.logp(Level.FINE, "CommonClassLoaderManager",
+            logger.logp(Level.SEVERE, "CommonClassLoaderManager",
```

Comment:
What's with the increase in Log Level? Why is this SEVERE can an operator who receives the SEVERE log level figure out what to do to fix it?